test/acceptance/helpers.js

```diff
@@ -0,0 +1,95 @@
+import express from 'express'
+import Nightmare from 'nightmare'
+import moment from 'moment'
+import cheerio from 'cheerio'
+import url from 'url'
+
+const {
+  ACCEPTANCE_TIMEOUT,
+  API_URL,
+  APP_URL,
+  METAPHYSICS_ENDPOINT,
+  POSITRON_URL
+} = process.env
+const [
+  FORCE_PORT,
+  GRAVITY_PORT,
+  METAPHYSICS_PORT,
+  POSITRON_PORT
+] = [
+  APP_URL,
+  API_URL,
+  METAPHYSICS_ENDPOINT,
+  POSITRON_URL
+].map((u) => url.parse(u).port)
```

Comment:
It's taking the ports out of the url environment variables—there's probably a more straightforward way to do this. Maybe just copy + pasting `const FORCE_PORT = url.parse(APP_URL).port`